### DISCUSSION TITLE
What do you want in new update

### DISCUSSION THREAD

**Discussion by sheikhhaziq** [sheikhhaziq/gyawun_music#47] [Ideas]
Write down the features you want in new update of this app.

**Comment by owenconnorz** [comment]
youtube music home page design and if login would be possible it will show better recommendations for what u listen to.

Downloads.

Add to Playlist.

Alexa and google home cast support.

**Comment by Silther** [comment]
Do not distort the appearance of thumbnails etc.. Instead, crop them to the new aspect ratio.

**Reply by sheikhhaziq** [reply]
Sure

**Comment by senolfndk** [comment]
settings and move the search box to the top right corner

**Comment by GET100PERCENT** [comment]
I am waiting for the  "Start Radio" button to be implemented in Vibe Music.

**Comment by GithubUser452** [comment]
Show new songs published by library artists

**Reply by sheikhhaziq** [reply]
This feature will be added soon.

**Comment by kunal1320k** [comment]
Lyrics

**Reply by sheikhhaziq** [reply]
Lyrics support is already added.

**Comment by dylanisaiahp** [comment]
I would like the ability to set the nav bar to the bottom, instead of the side

**Comment by Mickael81** [comment]
I would like to be able to translate "Gyawun Music" into another language by using Weblate or Crowdin.
